### DISCUSSION TITLE
有没有微信qq交流群？

### DISCUSSION THREAD

**Discussion by jilieryuyi** [milvus-io/milvus#7519] [Q&A and General discussion] [answered]
如题

**Comment by shiyu22** [comment, marked as answer]
You can scan this QR code: https://github.com/milvus-io/bootcamp/blob/master/images/wechat_group.png

**Reply by gobigrassland** [reply]
我扫描加了Zilliz 小助手，并没有将我拉入技术讨论群，是没有人来维护这个事情了吗？
